assets/css/style.css

```
@@ -122,5 +185,4 @@ button {
 .wrapper {
   max-width: 1600px;
   margin: 0 auto;
-}
-
+}
```

```suggestion
}

```
and place .wrapper under the body